File: dodam-core/src/main/java/b1nd/dodamcore/member/repository/BroadcastClubMemberRepository.java
package b1nd.dodamcore.member.repository;

import b1nd.dodamcore.member.domain.entity.BroadcastClubMember;
import b1nd.dodamcore.member.domain.entity.Member;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BroadcastClubMemberRepository extends JpaRepository<BroadcastClubMember, Integer> {

    boolean existsByMember(Member member);

}

File: services/service-member/src/main/kotlin/com/b1nd/dodamdodam/member/domain/broadcastclub/repository/BroadcastClubMemberRepository.kt
package com.b1nd.dodamdodam.member.domain.broadcastclub.repository

import com.b1nd.dodamdodam.member.domain.broadcastclub.entity.BroadcastClubMemberEntity
import com.b1nd.dodamdodam.member.domain.member.entity.MemberEntity
import org.springframework.data.jpa.repository.JpaRepository

interface BroadcastClubMemberRepository : JpaRepository<BroadcastClubMemberEntity, Long> {
    fun findByMember(member: MemberEntity): BroadcastClubMemberEntity?
    fun existsByMember(member: MemberEntity): Boolean
}
